test/parsers.js

```diff
@@ -381,6 +381,12 @@ test('#parseError()', function (t) {
       _conf: {
         sourceLinesErrorAppFrames: 5,
         sourceLinesErrorLibraryFrames: 5
+      },
+      logger: {
```

Comment:
Consider DRY'ing this up (is the DRY ruby'ism known in outside of the Ruby world?)

Reply:
fixed